src/SectionsList.tsx

```diff
@@ -15,11 +15,19 @@ const SectionsList: React.FC<Props> = ({
   displayDescription,
   displayNotes
 }) => {
+  if(!sections) {
+    return(<div></div>)
+  }
   return (
     <div>
       {sections.map((section, i) => (
         <div css={{ padding: "10px" }} key={section.registrationNumber}>
           <Section isOdd={!!(i % 2)} {...section} />
+          <SectionsList 
```

Comment:
How about we do `{section.recitations && <SectionsList sections={section.recitations}... /> }

Reply:
Got it

Reply:
Will merge again, but please format w/prettier